devenv/docker/blocks/slow_proxy_mac/main.go

```diff
@@ -0,0 +1,31 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"net/http/httputil"
+	"net/url"
+	"os"
+	"time"
+)
+
+func main() {
+	origin := os.Getenv("ORIGIN_SERVER")
+	if origin == "" {
+		origin = "http://localhost:9090/"
```

Comment:
Lets also change this default to: http://host.docker.internal:9090/

Reply:
👍